src/hooks/tests/useToggle.test.tsx

```diff
@@ -19,7 +20,7 @@ const resetHandlerMocks = () => {
 };
 
 // eslint-disable-next-line react/prop-types
```

Comment:
nit/unrelated suggestion: could opt to disable prop-types rule for `*.test.*` files so we don't need to have these eslint disable comments throughout test files.

Reply:
Sure, but I'm gonna leave that out of this PR for now.